From 9a7ba0a443bf92bf7dabdcbb2f7eb282991d96cb Mon Sep 17 00:00:00 2001
From: Jose Dapena Paz <jose.dapena@lge.com>
Date: Tue, 4 Jun 2019 22:01:03 +0200
Subject: IWYU: ThemeService requires NativeTheme

As ThemeService referes to NativeTheme through a ScopedObserver,
the full declaration is required.

Bug: 819294
Change-Id: I9d5bd2e87cfaa76e87f9b5509daea24848906a63

diff --git a/chrome/browser/themes/theme_service.h b/chrome/browser/themes/theme_service.h
index 6c79c723cd8c..f93dc0dbd490 100644
--- a/chrome/browser/themes/theme_service.h
+++ b/chrome/browser/themes/theme_service.h
@@ -25,6 +25,7 @@
 #include "extensions/buildflags/buildflags.h"
 #include "extensions/common/extension_id.h"
 #include "ui/base/theme_provider.h"
+#include "ui/native_theme/native_theme.h"
 #include "ui/native_theme/native_theme_observer.h"
 
 class BrowserThemePack;
-- 
2.20.1

